### ISSUE TITLE
README "Instructions for First Use" does not work in Windows

### ISSUE THREAD

**Issue by ojnas** [Telecominfraproject/oopt-gnpy#294]
The README instructions were recently changed, now telling the user to run the transmission main example like this:

`$ ./examples/transmission_main_example.py`

i.e. without typing `python` before the name of the script. This works in linux because of the shebang in `transmission_main_example.py` but it does not work in Windows unless the "[Python Launcher for Windows](https://docs.python.org/3/using/windows.html#python-launcher-for-windows)" is used to open .py files by default. Even when this launcher is used it does not work if you are in a virtual environment created with `python -m venv`. In this case it seems the [shebang](https://docs.python.org/3/using/windows.html#shebang-lines) actually tells the python launcher to use the default python interpreter instead of the one in the virtual environment resulting in a confusing ModuleNotFoundError.

**Comment by jktjkt** [comment]
Good catch, thanks. I'll change this so that the invocation is explicit via `python ./examples/transmission_main_example.py`.

**Comment by ojnas** [comment]
Digging a bit deeper... It seems the python launcher is now the "recommended" way to invoke python on Windows. Unfortunately this does not work with a version-specific shebang (i.e. python3 as we have in both transmission_main_example.py and path_requests_run.py) when working in a virtual environment, see e.g. discussion [here](https://bugs.python.org/issue28686). Maybe we should consider changing to:

`#!/usr/bin/env python`

in order to be portable between Linux and Windows.
